modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/service/ModelService.java

```diff
@@ -28,9 +28,9 @@
 
     List<ModelDto> getModels(String metricKey);
 
-    void addModelParams(ModelParams modelParams);
+    void addModelParams(String modelUUID, Hyperparams hyperparams);
```

Comment:
Is `addModelParams` the right name for this method? (Please see the comments from the last code review around this point.)